src/components/Tooltip/Tooltip.js

```diff
@@ -12,9 +12,24 @@ const propTypes = {
   content: PropTypes.string,
 };
 
-const Tooltip = ({ content = '', children }) => {
+function useCombinedRefs(forwardedRef) {
+  // this is mainly for 'CustomElement' because they might need to update the DOM when rendering 
+  const targetRef = useRef();
+
+  useEffect(() => {
+    if (!forwardedRef) {
+      return
+    }
+
+    forwardedRef.current = targetRef.current;
+  }, [forwardedRef]);
+
+  return targetRef;
+}
+
+const Tooltip = forwardRef( ({ content = '', children }, forwardedRef) => {
   const timeoutRef = useRef(null);
-  const childRef = useRef(null);
+  const childRef = forwardedRef ? useCombinedRefs(forwardedRef) : useRef(null);
```

Comment:
Can you explain more about why `useCombinedRefs` is necessary here? I tried to remove this, and just having

```javascript
const childRef = forwardedRef ? forwardedRef : useRef(null);
```

also works.

Reply:
Thank you for pointing this out. I was using an example I saw online but I guess we didn't need it after all